vllm/config.py

```diff
@@ -379,6 +381,8 @@ def __init__(
         self.max_num_seqs = max_num_seqs
         self.max_model_len = max_model_len
         self.max_paddings = max_paddings
+        self.policy = policy
+        self.max_delay = max_delay
```

Comment:
Perhaps a better name for `max_delay` (`order_window` or something similar)?
Could you add comments in the Args for the two new input variables as well?